docs/en/observability/apm/manage-storage/ilm-how-to.asciidoc

```diff
@@ -8,133 +8,123 @@
 :append-legacy:
 // tag::ilm-integration[]
 
-Index lifecycle policies allow you to automate the
-lifecycle of your APM indices as they grow and age.
-A default policy is applied to each APM data stream,
-but can be customized depending on your business needs.
-
-See {ref}/index-lifecycle-management.html[{ilm-init}: Manage the index lifecycle] to learn more.
+Lifecycle policies allow you to automate the lifecycle of your APM indices as they grow and age.
+A default policy is applied to each APM data stream, but can be customized depending on your business needs.
 
 [discrete]
 [id="index-lifecycle-policies-default{append-legacy}"]
 == Default policies
 
-The table below describes the default index lifecycle policy applied to each APM data stream.
-Each policy includes a rollover and delete definition:
+[NOTE]
+====
+As of 8.15, by default new clusters use {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide data retention settings for APM data.
```

Comment:
```suggestion
As of 8.15, new clusters use {ref}/data-stream-lifecycle.html[data stream lifecycle (DSL)] to provide the default data retention settings for APM data. The customization for lifecycle is still performed using ILM policies which are now configured to override the default DSL polices.

For clusters upgrading to 8.15, the default ILM policies should still work as expected. Users can switch the default to DSL explicitly by using the datastream API.
```

Something like this maybe? Just want to convey that the customization options are still there and the special handling for upgrades. Also, do you think we should refer to the known-issues doc: https://www.elastic.co/guide/en/observability/current/apm-known-issues.html#_upgrading_to_v8_15_0_may_cause_apm_indices_to_lose_their_lifecycle_policy?

Reply:
I think it's a good idea to explain how to switch to ILM. Should we also mention that for simpler changes to the retention duration, one can customise that through a `@custom` component template?

Reply:
> Should we also mention that for simpler changes to the retention duration, one can customise that through a @custom component template?

Datastreams won't take changes from component templates for retention duration to update existing datastreams so we can't update the component templates to change the retention duration -- new datastreams will take the updated setting but for updating the existing datastreams we need to explicitly use the datastream update API.